src/parameters.jl

```diff
@@ -48,12 +51,25 @@ function flatten(x::Positive)
 end
 
 """
-    Bounded(value::Real, lower_bound::Real, upper_bound::Real)
+    bounded(value::Real, lower_bound::Real, upper_bound::Real)
 
+Constructs a `bounded`.
 The `value` of a `Bounded` is a `Real` number that is constrained to be within the interval
```

Comment:
It might just be me that confused about `value`. I think you mean the `value` as the function. At some point I thought it is the value as the first field in Bounded. Do you think it worths to change the first field name in Positive/Bounded to be `unconstrainted_value` (and maybe the equivalent in positive/bounded to be `constrained_value`?

Reply:
@wytbella could you let me know if this is more readable?

Reply:
looks great! thanks!